File: app/src/main/java/com/lazycouple/restapiclient/ui/contract/ResponseContract.java
package com.lazycouple.restapiclient.ui.contract;

import android.os.Bundle;

import com.lazycouple.restapiclient.ui.presenter.BasePresenter;
import com.lazycouple.restapiclient.ui.viewModel.ResponseViewModel;

/**
 * Created by noco on 2016-10-12.
 */
public interface ResponseContract {

    interface View {

    }

    interface Presenter extends BasePresenter {
        void init(Bundle bundle);
        ResponseViewModel getViewModel();
    }
}

File: app/src/main/kotlin/com/lazycouple/restapiclient/ui/contract/ResponseContract.kt
package com.lazycouple.restapiclient.ui.contract

import android.os.Bundle
import com.lazycouple.restapiclient.ui.presenter.BasePresenter

/**
 * Created by noco on 2016-10-12.
 */
interface ResponseContract {

    interface View

    interface Presenter : BasePresenter {
        fun init(bundle: Bundle)
    }
}
